src/main/java/mate/controller/MenuController.java

```diff
@@ -2,14 +2,16 @@
 
 import java.io.IOException;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-public class IndexController extends HttpServlet {
+@WebServlet(urlPatterns = {"/", "/menu"})
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/menu")
```
can you simplify this pattern?

Reply:
Yes sure, it was just for testing.
+ Corrected in new commit.